.gitignore

```diff
@@ -9,3 +9,6 @@ venv/
 *.pyc
 app/db.sqlite3
 db.sqlite3
+**/migrations/**
+!**/migrations
+!**/migrations/__init__.py
```

Comment:
No need to ignore migrations and remove them from pr. Oleksandr meant, you should not make any changes in the models. So you could just delete an additional migration and push without it